DESCRIPTION

```diff
@@ -2,16 +2,19 @@ Package: LPWC
 Version: 0.99.3
 Date: 2018-05-28
 Title: Lag Penalized Weighted Correlation for Time Series Clustering
-Author: Thevaa Chandereng <chandereng@wisc.edu>, Anthony Gitter
-Maintainer: Thevaa Chandereng <chandereng@wisc.edu>
+Authors@R: c(
+    person("Thevaa", "Chandereng", ,"chandereng@wisc.edu", c("cre", "aut")),
+    person("Anthony", "Gitter", ,"gitter@biostat.wisc.edu", "aut"))
 Description: Computes a time series distance measure for clustering based on weighted correlation and introduction of lags. The lags capture delayed responses in a time series dataset. The timepoints must be specified. T. Chandereng, A. Gitter (2018) <doi:10.1101/292615>.
-Depends: 
+Depends:
     R (>= 3.0.2)
-Suggests: testthat, knitr, rmarkdown
+Suggests: testthat, rmarkdown
 License: MIT + file LICENSE
 Encoding: UTF-8
 LazyData: true
 RoxygenNote: 6.0.1
 VignetteBuilder: knitr 
 URL: https://github.com/gitter-lab/LPWC
 BugReports: https://github.com/gitter-lab/LPWC/issues
+Imports: ggplot2,
+    knitr
```

Comment:
Did you want a newline here?

Reply:
I added it using devtools::use_package(), so it overwrites by itself

Reply:
@agitter, regarding your comment:

> lags are always with respect to the original times, not other genes.

I look at each genes as a time series, and my understanding is that lags are computed by comparison of one time series to another. While I get the point that it's with respect to the original times, what confuses me is which `gene` (time series) is being used as reference. Is it the one with zero lag?